packages/dds/tree/src/feature-libraries/sequence-field/format.ts

```diff
@@ -78,17 +78,12 @@ export interface LineageEvent {
     readonly offset: number;
 }
 
-export interface Insert extends HasTiebreakPolicy, HasRevisionTag {
-    type: "Insert";
-    content: ProtoNode[];
-}
-
-export interface ModifyInsert<TNodeChange = NodeChangeType>
+export interface Insert<TNodeChange = NodeChangeType>
     extends HasTiebreakPolicy,
         HasRevisionTag,
         HasChanges<TNodeChange> {
-    type: "MInsert";
-    content: ProtoNode;
+    type: "Insert";
+    content: ProtoNode[];
 }
 
 export interface MoveIn extends HasMoveId, HasPlaceFields, HasRevisionTag {
```

Comment:
I don't remember why we decided to put changes on both move-ins and move-outs, but I think it would be simpler to only have changes on move-outs if that's possible.

Reply:
Happy to try it.